specification/resources/resource-manager/Microsoft.Authorization/stable/2017-04-01/locks.json

```diff
@@ -0,0 +1,1089 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ManagementLockClient",
+    "version": "2017-04-01",
+    "description": "Azure resources can be locked to prevent other users in your organization from deleting or modifying resources."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "AuthorizationOperations_List",
+        "description": "Lists all of the available Microsoft.Authorization REST API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Need default response to handle error.
https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review